### PR TITLE
Improved language switcher

### LINKED ISSUE #303: Language menu has low contrast and is difficult to read over the typrewriter effect.

### PR DESCRIPTION
**Pull request by ClientCrash**
**Issue**
Closes #303

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,7 +21,8 @@
     "@tryghost/content-api": "1.5.11",
     "nuxt": "2.15.7",
     "nuxt-i18n": "6.28.1",
-    "nuxt-lazy-load": "1.2.6"
+    "nuxt-lazy-load": "1.2.6",
+    "vue-clickaway": "^2.2.2"
```
Comment (MarcelCoding) [inline package.json:25]: > Published 4 years ago
Ether we should use another liberty or copy/fork the code to our repo. (If you do so, please include a GitHub Link to the original project.)

Comment (MarcelCoding) [reply package.json:25]: ^^ @ClientCrash

File: components/LanguageSwitcher.vue [hunk 2]
```diff
@@ -51,24 +60,24 @@ export default {
...
+  font-size: 1.2rem;
+  background-color: black;
   border-radius: 0.25rem;
+  margin-right: 4vw;
 }
 
 .languageItem {
   padding: 5px;
-  color: var(--colors-background-c);
+  color: #50AF2F;
```
Comment (MarcelCoding) [inline components/LanguageSwitcher.vue:71]: I would use white rather than blue.

**Review by MarcelCoding — CHANGES_REQUESTED** [review]

**Comment by ClientCrash** [comment]
Todo:
- [ ] Return to old location
- [ ] set color to white

**Review by MarcelCoding — APPROVED** [review]

**Review by DevMimas — APPROVED** [review]

**Pull request merged**